### PR TITLE
Localization - Improve Simplified Chinese

### PR DESCRIPTION
**Pull request by CCRcmcpe**
**When merged this pull request will:**
- Improve Simplified Chinese translation
- **AND** fix some strings have no correct localization
- Other enhancements related to stringtables

### IMPORTANT

- [ ] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3mod.com/).
- [x] [Development Guidelines](https://ace3mod.com/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/realisticnames/stringtable.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8 "?>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

**Comment by PabstMirror** [comment]
@mihuan-0 - let us know if you have any thoughts on these

**Comment by mihuan-0** [comment]
> @mihuan-0 - let us know if you have any thoughts on these

Very well. CCRcmcpe, perhaps Virgo, changed all the English punctuation to Chinese format😹

**Review by PabstMirror — APPROVED** [review]

**Comment by PabstMirror** [comment]
Is this ready?

**Comment by CCRcmcpe** [comment]
May complete in 3 days.

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**